app/inpage/ts/inpage.ts

```diff
@@ -431,7 +431,7 @@ class InterceptorMessageListener {
 			this.WindowEthereumRequest({ method: 'signer_chainChanged', params: [chainId] })
 		})
 
-		this.connected = window.ethereum.isConnected()
+		this.connected = window.ethereum.isConnected !== undefined ? window.ethereum.isConnected() : true
```

Comment:
If you wanted, you could simplify this to:
```suggestion
		this.connected = !window.ethereum.isConnected || window.ethereum.isConnected()
```